popcorn-official/popcorn-android
# Migrate to AndroidX

This PR migrates the project from legacy support libraries to AndroidX. It also updates the target/compile SDK to 29, Java version to 8, multidex library to latest and Picasso version to latest. RobotoTextView was removed from the project and replaced with plain TextView, since it appears to be incompatible with the latest version of appcompat (clashing attribute names).

Tests performed:
* Load main screen
* Go to movie details screen
* Play selected movie
* Verify subtitles, play/pause and fast forward/rewind work
* Repeat for 1-2 movies

Devices tested:
* Pixel 3XL (mobile)
* Samsung S10+ (mobile)
* Google ADT-3 (tv)

It is worth noting that no legacy devices were tested, because I don't have any handy. It would be helpful if someone could test this on devices running API level 21 (or lower) for mobile and tv.

Migrating to AndroidX somehow fixes the voice search for Android TV devices, which has been broken for some time. It's still not 100% reliable, but it works better than 0% of the time unlike previous versions.
